src/Command/MinifyTemplatesCommand.php

```diff
@@ -104,4 +113,23 @@ private function addAttributeOptionsToFamilyTemplate(array $familyTemplate, arra
 
         return $familyTemplate;
     }
+
+    private function addAttributeGroupsToFamilyTemplates(array $familyTemplates, array $attributeGroups): array
+    {
+        return array_map(
+            fn (array $familyTemplate) => $this->addAttributeGroupsToFamilyTemplate($familyTemplate, $attributeGroups),
+            $familyTemplates,
+        );
+    }
+
+    private function addAttributeGroupsToFamilyTemplate(array $familyTemplate, array $attributeGroups): array
```

Comment:
For me attribute group should not de defined at each attribute level but at family level like the attribute options